services/workflows-service/src/workflow/workflow.controller.external.ts

```diff
@@ -114,6 +115,7 @@ export class WorkflowControllerExternal {
   @UseKeyAuthGuard()
   @common.HttpCode(200)
   @swagger.ApiForbiddenResponse({ type: errors.ForbiddenException })
+  @UseFilters(new PrismaClientValidationFilter())
```

Comment:
Can you apply this filter globally? when thinking about it, must of the endpoint interacts with the db

Reply:
Its set